app/views/proposals/show.html.haml

```diff
@@ -83,7 +83,7 @@
                 = @event.room.name
           .col-md-12
             %dt Conference:
-            %dd= @event.program.conference.title
+            %dd= link_to @event.program.conference.title, conference_path(@conference.short_title)
```

Comment:
We shouldn't need to put `@conference.short_title`, just `@conference`, because of [`Conference#to_param`](https://github.com/openSUSE/osem/blob/master/app/models/conference.rb#L725-L727)... but that's just a nitpick.